CBLClient/Apps/JavaRequestHandlers/src/main/java/com/couchbase/mobiletestkit/javacommon/RequestHandler/DatabaseRequestHandler.java

```diff
@@ -144,7 +145,7 @@ public void updateDocument(Args args) throws CouchbaseLiteException {
     public void updateDocuments(Args args) throws CouchbaseLiteException {
         final Database database = args.get("database");
         final Map<String, Map<String, Object>> documents = args.get("documents");
-        database.inBatch(new Runnable() {
+        database.inBatch(new Fn.TaskThrows<RuntimeException>() {
```

Comment:
The way this code is intended to be used is with a lambda:

database.inBatch(() -> {
		for (Map.Entry<String, Map<String, Object>> entry: documents.entrySet()) {
				String id = entry.getKey();
				Map<String, Object> data = entry.getValue();
				MutableDocument updatedDoc = database.getDocument(id).toMutable();
				updatedDoc.setData(data);
				try { database.save(updatedDoc); }
				catch (CouchbaseLiteException e) { Log.e(TAG, "DB Save failed", e); }
		}
});